cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -1215,3 +1239,41 @@ func (k *Kuberhealthy) configureInfluxForwarding() {
 	}
 	k.MetricForwarder = metricClient
 }
+
+func listUnstructuredKHChecks() (*unstructured.UnstructuredList, error){
+
+	var unstructuredList *unstructured.UnstructuredList
+	restConfig, err := clientcmd.BuildConfigFromFlags("", configPath)
+	if err != nil {
+		return unstructuredList, err
+	}
+
+	dynClient, err := dynamic.NewForConfig(restConfig)
+	if err != nil {
+		return unstructuredList, err
+	}
+
+	khCheckGroupVersionResource := schema.GroupVersionResource{
+		Version: checkCRDVersion,
+		Resource: checkCRDResource,
+		Group: checkCRDGroup,
+	}
+
+	unstructuredList, err = dynClient.Resource(khCheckGroupVersionResource).Namespace("").List(metav1.ListOptions{})
+	if err != nil {
+		return unstructuredList, err
+	}
+
+	return unstructuredList, err
+}
+
+func convertUnstructuredKhCheck(unstructured unstructured.Unstructured) (khcheckcrd.KuberhealthyCheck, error){
+	un := unstructured.UnstructuredContent()
+	var khCheck khcheckcrd.KuberhealthyCheck
+	err := runtime.DefaultUnstructuredConverter.FromUnstructured(un, &khCheck)
+	if err != nil {
+		return khCheck, errors.New("Error converting unstructured object to khcheck: " + err.Error())
```

Comment:
next time check out the new-ish `fmt.Errorf` function and use the `%w` variable to wrap the error in your new error... This way you can use the new `errors.Is` call to improve error handling.